scripts/makefile/front.mk

```diff
@@ -35,6 +35,7 @@ front-build:
 		$(call frontexec, node -v); \
 		$(call frontexec, yarn -v); \
 		$(call frontexec, yarn build --verbose); \
+		$(call frontexec, yarn build-storybook --verbose); \
```

Comment:
It's not a good idea to add this command here : `front-build` job should remain as it is and contain `yarn build` only

It would be better to : 
- either create a new job to in [`deploy`](https://github.com/skilld-labs/skilld-docker-container/blob/195-add-static-storybook/.gitlab-ci.yml#L26) CI step 
   - Advantage is that job can be set as "manual" , and/or "always" only for master
- either add `make storybook` inside of `make all_ci` in [Makefile](https://github.com/skilld-labs/skilld-docker-container/blob/195-add-static-storybook/Makefile#L35)
- In both case, please use `make storybook` instead of `$(call frontexec, yarn build-storybook --verbose);` anyways